src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Link.js

```diff
@@ -34,9 +38,53 @@ export default class Link extends React.Component<FieldTypeProps<LinkValue>> {
             },
         } = this.props;
 
+        if (enableAnchor !== undefined && enableAnchor !== null && typeof enableAnchor !== 'boolean') {
+            throw new Error('The "enable_anchor" schema option must be a boolean if given!');
+        }
+
+        let enableTarget = false,
+            enableTitle = false,
+            enableRel = false;
+
+        if (enableAttributes !== undefined && enableAnchor !== null) {
+            if (typeof enableAttributes !== 'boolean') {
```

Comment:
the description of this pr passes a collection here. is the description outdated?

Reply:
yes, I adjusted it